src/ensembl/src/content/app/browser/drawer/drawer-views/gene-summary/GeneSummary.scss

```diff
@@ -38,10 +37,35 @@
 .featureSymbol {
   font-weight: $bold;
 }
-.stableId{
+.stableId {
   font-weight: $bold;
 }
-.featureSymbol + .stableId{
+.featureSymbol + .stableId {
   margin-left: 12px;
   font-weight: $light;
 }
+
+.downloadRow {
+  > .value {
+    display: flex;
+    position: relative;
+    width: 400px;
```

Comment:
I tried to remove `position:relative` and `width:400px` from the declaration block, and couldn't see any difference in the layout. Are these two rules needed for anything?

Reply:
Think the styling here and in the comment mentioned below are obsolete. I'll remove them.

Reply:
✅